src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -229,7 +231,10 @@ async def create_privacy_request(
                 )
                 created.append(privacy_request)
                 continue  # Skip further processing for this privacy request
-
+            if config.notifications.send_request_receipt_notification:
+                _send_privacy_request_receipt_email_to_user(
+                    db, policy, privacy_request_data.identity.email
+                )
```

Comment:
This seems like it should come before the subject identity verification send?  If `subject_identity_verification_required` is True, it will exit before this point and be resumed through a different endpoint and never receive this email.

Reply:
good catch on this. I'm actually unsure from a product perspective which should come first. Either way, I'll have to update this code so it works properly. @seanpreston what are your thoughts?

Reply:
Thanks for double-checking — the subject identity verification should always come first. Fidesops should always be verifying email addresses before sending email to them.